src/lib/containers/widgets/query-filter/EnumFacetFilter.tsx

```diff
@@ -308,35 +299,46 @@ export const EnumFacetFilter: React.FunctionComponent<EnumFacetFilterProps> = ({
     </div>
   )
 
-  // Any click event for the Dropdown will close the dropdown (assuming its open), so we have
-  // to handle the onToggle event and manually manage the dropdown open state. If metadata
-  // is defined the event occuring is inside the dropdown which we then want to keep open, otherwise
-  // we close it.
-  const onToggle = (_show: boolean, _event: any, metadata: any) => {
-    if (metadata.source) {
-      setIsShowDropdown(true)
-    } else {
-      setIsShowDropdown(false)
-    }
-  }
+  const onToggle = () => setIsShowDropdown(!isShowDropdown)
```

Comment:
The old `onToggle` seemed to cause more problems than it solved. I couldn't get the filter menus to close under any circumstances. It was much easier to toggle the open state by clicking the dropdown.